### ISSUE TITLE
HOW SET PROGRESS PROGRAMMATICALLY?

### ISSUE THREAD

**Issue by aliaksandr-chaulytka** [syedowaisali/crystal-range-seekbar#121]
HOW SET PROGRESS PROGRAMMATICALLY?
like on a picture
<img width="475" alt="screenshot_2018-06-05-23-00-31" src="https://user-images.githubusercontent.com/20597176/40999790-a08a69e0-6914-11e8-89c3-00a75ad9fe5a.png">
fail, there can be no question of a star

**Comment by MarinaShaposhnikova** [comment]
@memento2801 
What kind of problems do you have with it?
You should call
seekbarRange.setMinStartValue(minValue)
seekbarRange.setMaxStartValue(maxValue)

and then
seekbarRange.apply()

**Comment by ejdrian313** [comment]
add this to documentation, because apply to seekbar is so unintuitive. I lost hour to find this..

**Comment by keehoo** [comment]
@MarinaShaposhnikova so, the MinStartValue and maxStartValue are the current values, right? If so.. the name should be corrected imho, as for me it took a while to understand that, perhaps (as per android SDK Seekbar minProgress and maxProgress).

**Comment by MarinaShaposhnikova** [comment]
@keehoo No, the current values are minValue and maxValue if I remember correctly.
I'm not the owner of the library, I  only used it some months ago :)

**Comment by idrisadetunmbi** [comment]
@MarinaShaposhnikova, what identifies the range values (not current values) then?

**Comment by tayyabejaz** [comment]
@MarinaShaposhnikova setMinStartValue() doesn't set the slider to the specific position. why is it so?
@syedowaisali
